src/components/Explore/utils.ts

```diff
@@ -285,3 +290,29 @@ export function weeksAgo(dateFrom: Date, dateTo: Date) {
     return `${weeksAgo} ${daysAgo} ago`;
   }
 }
+
+export function getLocationLabel(location: Location) {
+  return location.county
+    ? `${location.county}, ${location.state_code}`
+    : location.state;
+}
+
+export function getLocationNames(locations: Location[]) {
+  return locations.map(getLocationLabel).join(', ');
+}
+
+const isCounty = (location: Location) => location.county;
```

Comment:
Consider `location.county !== undefined` so that this returns bool as the name suggests it does?  (or if you're into brevity, `!!location.county` works too)